.github/workflows/golangci-lint.yml

```diff
@@ -14,9 +14,13 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.16.0'
+          check-latest: false
       - uses: actions/checkout@v2
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
```

Comment:
Let's leave a comment that we can't upgrade this unless we also upgrade to golang 1.19?